sqlglot/dialects.py

```diff
@@ -706,6 +706,7 @@ class SQLite(Dialect):
         exp.DataType.Type.FLOAT: "REAL",
         exp.DataType.Type.DOUBLE: "REAL",
         exp.DataType.Type.DECIMAL: "REAL",
+        exp.DataType.Type.ORA_NUMBER: "REAL",
```

Comment:
delete this, 

oracle should just have a mapping of "NUMBER" -> TokenType.FLOAT